scripts/jenkins/ardana/ansible/repositories.yml

```diff
@@ -75,8 +55,8 @@
       - Cloud
       - SLES12-SP3-Pool
       - SLES12-SP3-Updates
-      - SUSE-OpenStack-Cloud-8-Pool
-      - SUSE-OpenStack-Cloud-8-Updates
+        # - SUSE-OpenStack-Cloud-8-Pool
+        # - SUSE-OpenStack-Cloud-8-Updates
```

Comment:
Why comment these out instead of removing them?

Reply:
I guess we need to conditionalize it at some point so the repository names are still useful imho